File: galgamecommon/src/main/java/com/goexp/galgame/common/website/WikiURL.java
package com.goexp.galgame.common.website;

import com.goexp.common.util.UrlBuilder;

public final class WikiURL {

    public static final String searchUrl = "https://ja.wikipedia.org/w/index.php";

    public static String fromTitle(String title) {
        return UrlBuilder
                .create(searchUrl)
                .param("search", title)
                .build();
    }

}

File: galgamecommon/src/main/scala/com/goexp/galgame/common/website/WikiURL.scala
package com.goexp.galgame.common.website

import com.goexp.common.util.web.url.UrlBuilder

object WikiURL {
  private val searchUrl = "https://ja.wikipedia.org/w/index.php"

  def fromTitle(title: String): String =
    UrlBuilder(searchUrl)
      .param("search", title)
      .build
}